scripts/pi-hole/php/auth.php

```diff
@@ -11,7 +11,6 @@
 if (empty($ERRORLOG)) {
     $ERRORLOG = '/var/log/lighttpd/error.log';
 }
-$regexfile = "/etc/pihole/regex.list";
```

Comment:
The [`add_regex`](https://github.com/pi-hole/AdminLTE/blob/483a3aee27d12cc54109778aba8f0eff737d2c97/scripts/pi-hole/php/func.php#L48) function can also be removed now, as it probably isn't used by anything anymore.